cmd/cloud-endpoints-controller/main.go

```diff
@@ -364,7 +389,12 @@ func calcParentSig(parent *CloudEndpoint, addStr string) string {
 	return fmt.Sprintf("%x", hasher.Sum(nil))
 }
 
-func makeDefaultOpenAPISpec(endpoint, target string, jwtAudiences []string) (string, error) {
+func validateOpenAPISpec(specOriginal string) error {
+	var spec map[string]interface{}
+	return yaml.Unmarshal([]byte(specOriginal), &spec)
+}
+
+func getWildcardAPITemplate() string {
 	templateString := `
```

Comment:
Remove this variable and newline after the opening of the block string. Just return the block string without creating an intermediate variable.